relay-general/src/protocol/contexts/profile.rs

```diff
@@ -0,0 +1,83 @@
+use crate::types::{Annotated, Error, FromValue, Value};
+use once_cell::sync::OnceCell;
+use regex::Regex;
+
+#[derive(Clone, Debug, Default, PartialEq, Empty, IntoValue, ProcessValue)]
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub struct ProfileId(pub String);
```

Comment:
In other places, `profile_id` is declared as `EventId`, which would make the custom `FromValue` implementation unnecessary AFAIK.

Reply:
See for example https://github.com/getsentry/relay/blob/37f965b9747fd00ba4f3b01be378279bb6704687/relay-profiling/src/android.rs#L37.